41-vaccine-temp-monitor/README.md

```diff
@@ -0,0 +1,321 @@
+Vaccine Temperature Monitor
+===========================
+
+The shelf life of vaccines (and other temperature sensitive medicine), can be
+prolonged by monitoring temperature and location using an ESP32, Notecard, and
+LM75A sensor. When an alert condition is sensed (the temperature exceeds a
+predefined threshold), this device will send a Twilio SMS alert with the exact
+location of the vaccine, so it can be rescued before it is ruined.
+
+**Feature Tags:**
+|ESP32|Arduino|Wi-Fi Triangulation|GPS (GNSS)|Tracking|Low-Power|
+|Environment Variables|SMS Alerts|
+
+Table of Contents
+-----------------
+
+- [Vaccine Temperature Monitor](#vaccine-temperature-monitor)
+  - [Table of Contents](#table-of-contents)
+  - [Prerequisites](#prerequisites)
+    - [Hardware](#hardware)
+    - [Software](#software)
+  - [Notehub Setup](#notehub-setup)
+    - [Twilio Route](#twilio-route)
+  - [Hardware Setup](#hardware-setup)
+  - [Notecard Firmware Setup](#notecard-firmware-setup)
+  - [Host Firmware Setup](#host-firmware-setup)
+  - [Environment Variables](#environment-variables)
+  - [Operation](#operation)
+    - [Normal Operation](#normal-operation)
+    - [Alarm Conditions](#alarm-conditions)
+      - [Thermal Event](#thermal-event)
+        - [Thermal Event Operation](#thermal-event-operation)
+      - [Low Battery](#low-battery)
+  - [Related Topics](#related-topics)
+
+Prerequisites
+-------------
+
+### Hardware
+
+- USB type A-to-micro USB cable
+- [2000 mAh Li-Ion battery with JST cable](https://www.adafruit.com/product/2011)
+- [Notecarrier-B](https://shop.blues.io/collections/notecarrier/products/notecarrier-b)
+- Molex Antenna 213353 (included with Notecarrier-B purchase)
+- [LTE Cat-1 Notecard (North America, EMEA)](https://shop.blues.io/collections/notecard)
+- [Adafruit Huzzah32 - ESP32 Feather Board (loose headers)](https://www.adafruit.com/product/3405)
+- [HiLetgo 2pcs LM75 LM75A Temperature Sensor Module](https://www.amazon.com/gp/product/B082KLD7XT/)
+- [Molex Antenna 209142](https://shop.blues.io/collections/accessories/products/flexible-cellular-or-wi-fi-antenna)
+- [Qwiic Cable Breakout](https://www.sparkfun.com/products/14425)
+- [[OPTIONAL] Cirocomm 5cm Ceramic Active GPS Antenna 32db (High Gain)](https://www.amazon.com/gp/product/B078Y2WNY6/)
+- [[OPTIONAL] ESP-Prog](https://www.digikey.com/en/products/detail/espressif-systems/ESP-PROG/10259352)
+
+### Software
+
+- [Visual Studio Code](https://code.visualstudio.com/)
+- [PlatformIO](https://platformio.org/)
+
+Notehub Setup
+-------------
+
+Sign up for a free account on [notehub.io](https://notehub.io) and
+[create a new project](https://dev.blues.io/quickstart/notecard-quickstart/notecard-and-notecarrier-pi/#set-up-notehub).
+
+### Twilio Route
+
+When an alert condition is detected a Note is immediately sent to Notehub. In
+order for you to be informed this event occurred, a Twilio route has been
```

Comment:
I would rephrase this. Just reads weird with passive voice. Suggestion:

In order for you to be informed this event occurred, you must configure a Twilio route.